src/CMakeLists.txt

```diff
@@ -58,14 +58,17 @@ set(common-external-libs
 	${Boost_IOSTREAMS_LIBRARY}
 	${Boost_REGEX_LIBRARY}
 	${Boost_PROGRAM_OPTIONS_LIBRARY}
-)
-
-set(common-external-libs
-	${common-external-libs}
 	${Boost_FILESYSTEM_LIBRARY}
 	${Boost_LOCALE_LIBRARY}
 )
 
+if(NOT WIN32 AND NOT CMAKE_CROSSCOMPILING AND NOT APPLE)
```

Comment:
It would be easier to just check [CMAKE_SYSTEM_NAME](https://cmake.org/cmake/help/v3.0/variable/CMAKE_SYSTEM_NAME.html).